configuration/drugs/drugs.csv

```diff
@@ -287,35 +296,37 @@ c109ae6a-9002-43ce-87c8-b1d29d5dd661,,Eucalyptus & Diphenhydramine & Phenylephri
 637499ff-1c54-4722-ba57-3e41b3d22a49,,Permethrin Cream 5%,6315c226-01f6-4c59-9332-74f5347c3ef7,0689bb6e-9371-4a0f-ad1c-63c135b1714d,,
 17f3e82f-279b-4657-bcb9-6361ad59e0e7,,Silver Nitrate Pencil 75%,984e4058-d87f-489f-a28d-138c90565666,0689bb6e-9371-4a0f-ad1c-63c135b1714d,75%,
 10a9b381-7926-459c-94ff-c81440430fbf,,Acide Ascorbique (Vit C) 1000mg,b7bab21f-5d84-4e48-b056-bdc25e696365,fdbe907b-6abc-41dc-9599-2d84105e4063,1000mg,
-9241d8ee-2fa4-4965-a6ca-8426beffe8a2,,Acide borique & Antipyrine & Ephedrine 33.65mg/ml,a546446e-e789-4d02-b02f-c938c6bd373f,7c46524c-a14f-43b6-b4b7-792cee31d8d5,33.65mg/ml,
-de4c409b-7d37-48a7-aecc-302189d4ab03,,Acide borique & Antipyrine & Benzocaine 80mg/ml,07470e7f-f959-4344-89f6-8e6a0f442bef,7c46524c-a14f-43b6-b4b7-792cee31d8d5,80mg/ml,
-54e2bf22-e7dc-48d5-9ee1-8132e1a354c7,,Acide tranexamique 1g/4ml,c6557538-cc7e-4ca8-bfeb-80ff5c4f4976,bd1b2599-84d2-4f33-ba25-4f6d42d147a5,1g/4ml,
+9241d8ee-2fa4-4965-a6ca-8426beffe8a2,,"Acide borique, Antipyrine, Ephedrine (ophtalnol) 33.65mg/ml",a546446e-e789-4d02-b02f-c938c6bd373f,7c46524c-a14f-43b6-b4b7-792cee31d8d5,33.65mg/ml,
+de4c409b-7d37-48a7-aecc-302189d4ab03,,"Acide borique, Antipyrine, Benzocaine (othorinol) 80mg/ml",07470e7f-f959-4344-89f6-8e6a0f442bef,7c46524c-a14f-43b6-b4b7-792cee31d8d5,80mg/ml,
```

Comment:
We should keep consistency. Don't we use `&` to separate active ingredients?

Reply:
> We should keep consistency. Don't we use `&` to separate active ingredients?

This has been fixed.